src/attestation/mod.rs

```diff
@@ -1165,6 +1132,11 @@ mod tests {
         };
 
         divergent_subgraphs.insert(comparison_result.deployment.clone());
-        notify(&mut divergent_subgraphs, &comparison_result).await;
+        if divergent_subgraphs.contains(&comparison_result.deployment) {
```

Comment:
When we make unit tests, it would be great if the actual functions can be tested, as in we make up the inputs or struct to test the output of the function. I think tests without asserts are not maintainable and not as strong as it could be even if the logic is the same at the moment.

Reply:
I agree, but in this case we would need to test the notifications which are external http calls :/

Reply:
sure but I still don't think this is testing anything, you could check on the result from those notifications if there's any, or send to a mock server with some responses, or remove this test since it isn't testing much

Reply:
agreed, removed ✅